op-deployer/pkg/deployer/pipeline/opchain.go

```diff
@@ -91,10 +94,88 @@ func DeployOPChain(ctx context.Context, env *Env, bundle ArtifactsBundle, intent
 		DelayedWETHPermissionlessGameProxyAddress: dco.DelayedWETHPermissionlessGameProxy,
 	})
 
+	currentBlock, _ := env.L1Client.BlockNumber(ctx)
+	block, _ := env.L1Client.BlockByNumber(ctx, big.NewInt(int64(currentBlock)))
```

Comment:
Do we need to check the error returned from these two calls, or is there a reason it's safe to ignore them here?

Reply:
Good spot, I'll make a follow up pr to fix this.

Reply:
We should create some semgrep rules to catch these - I will look into that.